framework/src/actions/MeshOnlyAction.C

```diff
@@ -67,14 +69,92 @@ MeshOnlyAction::act()
   {
     TIME_SECTION("act", 1, "Writing Exodus");
 
-    ExodusII_IO exio(mesh_ptr->getMesh());
+    auto & output_mesh = mesh_ptr->getMesh();
+    ExodusII_IO exio(output_mesh);
 
     Exodus::setOutputDimensionInExodusWriter(exio, *mesh_ptr);
 
     // Default to non-HDF5 output for wider compatibility
     exio.set_hdf5_writing(false);
 
     exio.write(mesh_file);
+
+    // Check if extra element integers should be outputted to Exodus file
+    unsigned int n_eeid = output_mesh.n_elem_integers();
+    if (n_eeid > 0)
```

Comment:
I would remove this if because it wont let you do warnings on bad names specified in output list if there's no extra elem id on the mesh

Reply:
This is removed and now a warning message has been added at the end of the next if statement